app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -110,6 +117,20 @@ offerSearchSuggestions=Autocomplete search term as you type
 doNotTrackTitle=Do Not Track
 doNotTrack=Send a 'Do Not Track' header with browsing requests (requires browser restart)
 blockCanvasFingerprinting=Fingerprinting Protection (may break some websites)
+advancedSettings=Advanced Settings...
+advancedSettingsTitle=Advanced Settings for Brave Payments
+ledgerRecoveryTitle=Recover your Brave wallet
+ledgerRecoverySubtitle=Enter your recovery keys below
+ledgerRecoveryContent=The balance of the recovered wallet will be transferred to your new Brave wallet. The old wallet will still exist as an empty wallet.
+ledgerBackupTitle=Backup your Brave wallet
+ledgerBackupContent=Below, you will find the anonymized recovery keys that are required if you ever lose access to this computer. We recommend that you print or save these keys and store them in a safe place, like your local safe deposit box, or under your mattress. It's really up to you!
```

Comment:
nit: can we get rid of the `like your local safe deposit box, or under your mattress. It's really up to you!` part? if i were a hacker i would start looking under people's mattresses after this launched. :p